src/backend/expungeservice/models/helpers/charge_classifier.py

```diff
@@ -172,11 +165,27 @@ def _felony_class_c(level):
             return FelonyClassC
 
     @staticmethod
-    def _felony_class_b(level):
+    def _felony_class_b(level, statute):
         if level == "Felony Class B":
-            return FelonyClassB
+            if ChargeClassifier._crime_against_person(statute):
+                return PersonFelonyClassB
+            else:
+                return FelonyClassB
 
     @staticmethod
     def _felony_class_a(level):
         if level == "Felony Class A":
             return FelonyClassA
+
+    @staticmethod
+    def _crime_against_person(statute):
```

Comment:
It looks like this method returns `Optional[bool]` and thus is relying on the fact that None is falsey. We should just return `False`.